sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/aad/webapp/AADWebSecurityConfigurerAdapter.java

```diff
@@ -31,6 +33,7 @@
  */
 public abstract class AADWebSecurityConfigurerAdapter extends WebSecurityConfigurerAdapter {
 
+    private static final String DEFAULT_FAILURE_URL = "/login?error";
```

Comment:
Delete `DEFAULT_FAILURE_URL`, use `"/login?error"` directly, because it only used one time.